### DISCUSSION TITLE
Adding Show in Taskbar Boolean

### DISCUSSION THREAD

**Discussion by EiTaNBaRiBoA** [godotengine/godot-proposals#8466] [Platforms]
Adding ShowInTaskbar bool in Godot. If true the application will show in taskbar like normally, if set to false it will not show in taskbar. This allow when creating applications in Godot and the user decides to minimize it, he will not see the app in the taskbar.

To access it via code can be done by
```python
get_viewport.ShowInTaskbar = true/false
```

**Comment by Calinou** [comment]
- Closing in favor of https://github.com/godotengine/godot-proposals/issues/8467.
